string ades_name
KeyValPair[] preconditions
KeyValPair[] effects
MotionSequence[] motion_sequences
